docs/workshops/interviews/technical/workshop.md

```diff
@@ -0,0 +1,262 @@
+
+## What is your goal in a technical interview?
+
+Your goal in a technical interview is to show that you're going to be someone the interviewer wants to work with.
+
+Think about people you've enjoyed working with - what traits do they have? I'll suggest a few:
+
+* Friendly - no one likes someone getting angry with them!
+* Patient.
+* Thoughtful - jumping straight into your first idea doesn't make it likely that you're doing the best thing.
+* Open to feedback - if you have an idea that may help me, it's frustrating if I don't listen to it.
+* Communicative - if I'm stuck, and you don't know what I'm thinking, it's hard for you to participate.
+* Able to think through a problem.
+* Able to write code that works.
+
+Notice that **most of these are not technical traits**. While you're solving technical problems in a technical interview, most of the point of the interview is to see how you interact with people, and how you think.
+
+## What structures do technical interviews have?
+
+Not all technical interviews are the same, but they tend to have similar elements:
+
+* The interviewer describes a problem.
+* You're expected to ask some questions about the problem (they may not tell you this, but it is _always_ expected!)
+* You solve the problem.
+  * Maybe this will be pair programming where you're at a keyboard together, maybe you'll be given a laptop or editor like CodeWars to write in online, maybe you'll be given a whiteboard to write on with a pen, or maybe you'll just talk.
+  * You are allowed to ask questions, and even if you're writing, you should always talk about what you're writing.
+* The interviewer will ask you some questions about your solution.
+* The interviewer may change the problem a bit, and ask you to solve it (again, either by writing it down, or just by talking about it).
+
+You may be asked to solve one problem, or more than one, but ordinarily you'll spend somewhere between 15 and 60 minutes on each problem you're asked.
+
+The interviewer probably doesn't care what programming language you use (so you should probably use Javascript!), and will say if they do.
+
+TODO: Work out how to advise trainees get this information: _Some_ interviewers care a lot about details (like remembering every `;`, and that you remember whether a function is called `startsWith` or `beginsWith` or `hasPrefix`). Others don't. It's worth asking how precise you need to be.
+
+:::tip Remember
+The goal is not to get to a solution as fast as possible. The goal is to show that you're going to be a good person to work with.
+:::
+
+Let's work through an example interview question to get a feel for how this works:
```

Comment:
Actually reviewing the whole thing, I don't think a whole separate workshop would be necessary, since most of the points still apply.

I think it could be handled with some notes at the end comparing the different styles.